### PR TITLE
Fix home

### PR DESCRIPTION
**Pull request by jimchen5209**
Fix /home command that never worked in #283 

also deleted unused /spawn command

### CODE REVIEW

**Comment by jimchen5209** [comment]
- [ ] Check bed exist
- [ ] No Bed Message

File: src/main/kotlin/one/oktw/galaxy/command/commands/Home.kt [hunk 1]
```diff
@@ -47,23 +48,38 @@ class Home : Command {
...
                     delay(TimeUnit.SECONDS.toMillis(1))
                 }
                 withContext(player.server.asCoroutineDispatcher()) {
-                    player.requestTeleport(source.player.spawnPosition.x.toDouble(),source.player.spawnPosition.y.toDouble(),source.player.spawnPosition.z.toDouble())
+                    player.teleport(
+                        source.minecraftServer.getWorld(player.dimension),
+                        position.x,
+                        position.y,
+                        position.z,
+                        0.0F,
```
Comment (james58899) [inline src/main/kotlin/one/oktw/galaxy/command/commands/Home.kt:78]: Use player original rotation?

Comment (jimchen5209) [reply src/main/kotlin/one/oktw/galaxy/command/commands/Home.kt:78]: 不好吧
原版重生好像也是 0 0

Comment (jimchen5209) [reply src/main/kotlin/one/oktw/galaxy/command/commands/Home.kt]: TBD

**Comment by jimchen5209** [comment]
- [ ] Number is strange

**Comment by jimchen5209** [comment]
P.S. 根據原版神奇的計算方式 /spawn 如果離出生點太近不會有感覺

**Comment by jimchen5209** [comment]
Rewrite needed because respawn logic changed

**Pull request closed without merging**